### DISCUSSION TITLE
Section 5.26 - Buckets

### DISCUSSION THREAD

**Discussion by alexandreleitetrevisani** [chgasparoto/curso-aws-com-terraform#171] [Q&A]
Boa noite, professor.
Estou tendo este erro e não consigo solucioná-lo. Obrigado.

terraform init -backend=true -backend-config="backend.hcl"

Initializing the backend...
Initializing modules...
Downloading git::https://github.com/chgasparoto/terraform-s3-object-notification.git for logs...
- logs in .terraform/modules/logs
- logs.notification in .terraform/modules/logs/modules/notification
- logs.objects in .terraform/modules/logs/modules/object
Downloading git::https://github.com/chgasparoto/terraform-s3-object-notification.git for redirect...
- redirect in .terraform/modules/redirect
- redirect.notification in .terraform/modules/redirect/modules/notification
- redirect.objects in .terraform/modules/redirect/modules/object
Downloading git::https://github.com/chgasparoto/terraform-s3-object-notification.git for website...
- website in .terraform/modules/website
- website.notification in .terraform/modules/website/modules/notification
- website.objects in .terraform/modules/website/modules/object
╷
│ Error: validating provider credentials: retrieving caller identity from STS: operation error STS: GetCallerIdentity, https response error StatusCode: 403, RequestID: 5a700013-1a18-422f-9cbf-dcd766c1f511, api error InvalidClientTokenId: The security token included in the request is invalid.

**Comment by chgasparoto** [comment]
@alexandreleitetrevisani pelo erro (_**Error: validating provider credentials**_) que você compartilhou, tente verificar como está a configuração de autenticação do seu provider.

Você está usando o profile correto? 
Este profile tem permissões suficientes para rodar os comandos do seu código?
